src/libpriv/rpmostree-core.cxx

```diff
@@ -2335,7 +2344,9 @@ rpmostree_find_and_download_packages (const char *const *packages, const char *s
   g_autofree char *reposdir = g_build_filename (repo_root ?: source_root, "etc/yum.repos.d", NULL);
   dnf_context_set_repo_dir (ctx->dnfctx, reposdir);
 
-  if (!rpmostree_context_download_metadata (ctx, DNF_CONTEXT_SETUP_SACK_FLAG_SKIP_RPMDB,
+  if (!rpmostree_context_download_metadata (ctx, (DnfContextSetupSackFlags) 
```

Comment:
The following is the trail of nested functions affected by setting the DNF_CONTEXT_SETUP_SACK_FLAG_SKIP_FILELISTS flag:

- sort_replacements()
        - handle_override()
            - rpmostree_override_builtin_replace()
            - rpmostree_override_builtin_remove()
            - rpmostree_override_builtin_reset()
    - prepare_download_pkgs_txn()
        - osexperimental_handle_download_packages()
 
From what I can tell, these functions relate to overriding specific packages with ones from a remote source? For that reason, I don't think disabling filelists would be a problem since you would be specifiy a remote source, not a filename path as input. Not too sure about this one either though haha...